extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -77,6 +76,12 @@ class AmpLightbox extends AMP.BaseElement {
 
   /** @override */
   activate() {
+    /**  @private {function(this:AmpLightbox, Event)}*/
+    this.boundCloseOnClick_ = event => this.closeOnClick(event);
+    this.boundCloseOnEscape_ = event => this.closeOnEscape(event);
+    this.getWin().document.documentElement.addEventListener(
```

Comment:
We'd also want "ESC" processing for image lightbox. I'd say, let's get rid of `close` attribute entirely and add `keydown` handler in image lightbox - then they'd work consistently.